core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -71,14 +73,9 @@ oppia.controller('Dashboard', [
     $rootScope.loadingMessage = 'Loading';
     DashboardBackendApiService.fetchDashboardData().then(
       function(response) {
-        $scope.explorationsList = response.explorations_list.sort(
-          function(a, b) {
-            return (a.title === '' ? 1 :
-              b.title === '' ? -1 :
-              a.title < b.title ? -1 :
-              a.title > b.title ? 1 : 0);
-          }
-        );
+        $scope.explorationsList = (
```

Comment:
As discussed in meeting: Use of `orderBy` won't help here since we will still have to call the sorting function manually each time.